tests/test_auto_machine.py

```diff
@@ -41,8 +41,9 @@ def test_auto_machine():
 
     assert config_dict['machine']=="auto"
 
+    send_text = str(platform.system()) + " " + str(platform.release()) + ": This is an automated test message."
```

Comment:
I'm seeing a lotta code copy-pasted from test_whecho_simple.py try using imports to lessen the amount of copy-pasted code here.

please revise this file.

Reply:
send_text is easy to convert to a global variable in test_whecho_simple and import into test_auto_machine

My issue is that there is a lot of other copy-pasted code such as using Popen for running command line

Reply:
let me switch to the branch and take a look at it

Reply:
check commit [fdb8a6f](https://github.com/cvraut/whecho/pull/15/commits/fdb8a6f305d75d0b2f7bfc256ce7dc817d6e970b) for how I re-used code from test_whecho_simple. This file should be good now. Fix the other one too.